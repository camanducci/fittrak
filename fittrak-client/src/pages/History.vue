<template>
  <v-container>
    <v-layout row wrap>
      <v-flex xs12>
        <WorkoutHistoryToolbar />
        <WorkoutList />
      </v-flex>
    </v-layout>
  </v-container>
</template>

<script>
import WorkoutList from "@/components/workouts/WorkoutList";
import WorkoutHistoryToolbar from "@/components/workouts/WorkoutHistoryToolbar";

export default {
  name: "History",
  components: {
    WorkoutList,
    WorkoutHistoryToolbar
  }
};
</script>

<style scoped>
</style>
