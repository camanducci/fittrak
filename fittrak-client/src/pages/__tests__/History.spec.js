import { shallowMount } from "@vue/test-utils";

import History from "@/pages/History";

describe("History", () => {
  test("History page renders", () => {
    const app = shallowMount(History);
    expect(app.html()).toMatchSnapshot();
  });
});
