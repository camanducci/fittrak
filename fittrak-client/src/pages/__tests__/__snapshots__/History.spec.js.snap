// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`History History page renders 1`] = `
<v-container-stub tag="div">
  <v-layout-stub tag="div" row="" wrap="">
    <v-flex-stub tag="div" xs12="">
      Nothing to see here.
    </v-flex-stub>
  </v-layout-stub>
</v-container-stub>
`;
